### PR TITLE
Tests and translation

### PR DESCRIPTION
**Pull request by Filiposki54**
added translations and tests to the login and registration form

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -42,5 +42,26 @@
...
+  "delete": "Delete",
+
+  "signup": "Signup",
+  "begin":"Begin your journey as a volunteer",
+  "firstName": "First Name",
+  "lastName": "Last Name",
+  "email": "Email",
+  "phone": "Phone Number (Optional)",
+  "password": "Password",
+  "showPassword": "Show Password:",
```
Comment (KinWaj) [inline public/locales/en/translation.json:54]: why is there ':' and in ukrainian and russian translations doesn't have it ?

File: public/locales/pl/translation.json [hunk 2]
```diff
@@ -42,5 +42,26 @@
...
+  "delete": "Usuń",
+
+  "signup": "Zarejestruj się",
+  "begin":"Rozpocznij swoją przygodę jako wolontariusz",
+  "firstName": "Imię",
+  "lastName": "Nazwisko",
+  "email": "Email",
+  "phone": "Numer telefonu (opcjonalnie)",
+  "password": "Hasło",
+  "showPassword": "Pokaż hasło:",
```
Comment (KinWaj) [inline public/locales/pl/translation.json:54]: same here

File: src/Components/Signup/Signup.js [hunk 3]
```diff
@@ -72,72 +64,72 @@ const Signup = () => {
...
             <br/>
             <div className="button-group">
             <button className="signup-form_button" type="submit" disabled={isSubmitting}>
-              Sign Up
+              {t('signup')}
             </button>
             <div className="line-text">
               <hr />
-              <p>Or continue with your email address</p>
+              <p>{t('orContinueWith')}</p>
```
Comment (KinWaj) [inline src/Components/Signup/Signup.js:128]: should be something different like 'or', got Agata's permission with that :) 
needed new variable with 'or ' in other lang's

**Review by KinWaj — CHANGES_REQUESTED** [review]
small changes required

**Review by ayakiri — APPROVED** [review]
Fixed Kinga's notes, looks good to me

**Pull request merged**